### ISSUE TITLE
Create API project

### ISSUE THREAD

**Issue by dancingsushii** [amosproj/amos2024ss04-building-information-enhancer#39]
## User story
1. As a Backend Developer I want to access basic structure of 
2. I want / need {}
3. So that {}

## Acceptance criteria
* API project is created
* Basic file infrastructure is created
* Test request is executed successfully
* Basic flow of the project: database access in API project

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
